libvast/src/system/catalog.cpp

```diff
@@ -564,4 +659,190 @@ catalog(catalog_actor::stateful_pointer<catalog_state> self,
     }};
 }
 
+record catalog_state::status(status_verbosity v) const {
```

Comment:
Please make it so functions in corresponding hpp and cpp files appear in the same order. For example, everything down from here should go below the other member functions, not at the bottom of the file.

Reply:
Aligned order in https://github.com/tenzir/vast/pull/2745/commits/6070dee424e0d491fd578501a51c9a4a98e87dad